packages/redux-routine/package.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "@wordpress/redux-routine",
+	"version": "1.0.0",
+	"description": "Redux middleware for generator coroutines.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"redux",
+		"middleware",
+		"coroutine"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/redux-routine/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"dependencies": {},
```

Comment:
`@babel/runtime` should be listed here because tranpiled code contains references to it. Example: 
```js
import _Promise from "@babel/runtime/core-js/promise";
```